superset/utils/memoized.py

```diff
@@ -64,7 +64,9 @@ def __get__(
         if not self.is_method:
             self.is_method = True
         # Support instance methods.
-        return functools.partial(self.__call__, obj)
+        func = functools.partial(self.__call__, obj)
+        func.__func__ = self.func  # type: ignore
+        return func
```

Comment:
I wasn't quite able to figure out why this is necessary, but running doctests on memoized methods started failing on 3.9 due to the 
The only thing I found is this which seems like the same problem, but is said to affect Python 3.8, not 3.9: https://bugs.python.org/issue12790 Hacky workaround, but does the trick. I'll try to figure out how to solve this more elegantly before this gets merged

Reply:
weird!